client/src/components/data-container.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Graph from "./graph";
+import Table from "./table";
+
+const dataContainer = (graphData, tableData) => {
```

Comment:
The return statement is fine, but here you have `graphData` and `tableData` as props for the `dataContainer` component. I am not sure where it is used, but this would mean that this component's parent would have to pass the data down to this component. Instead, we want the data to be fetched here. However, as I said, don't worry about implementing the fetch, just worry about declaring some variables with data, for example:

```js
const graphData = ["point1", "point2",etc..]
```
_but this should be in the structure needed for each of your child components, `Graph` and `Table`._